be/src/exec/schema_scanner/schema_rowsets_scanner.cpp

```diff
@@ -78,7 +82,19 @@ Status SchemaRowsetsScanner::start(RuntimeState* state) {
 
 Status SchemaRowsetsScanner::_get_all_rowsets() {
     if (config::is_cloud_mode()) {
-        return Status::NotSupported("SchemaRowsetsScanner::_get_all_rowsets is not implemented");
+        // cloud tablets
+        std::vector<std::weak_ptr<CloudTablet>> tablets =
+                ExecEnv::GetInstance()->storage_engine().to_cloud().tablet_mgr().get_weak_tablets();
```

Comment:
NOTICE that, tablets are in LRU cache for cloud mode, which means not all tablets can be seen when we try to `select * from information_schema.rowsets`. 
It should be mentioned here.

Reply:
done